packages/backend-deployer/src/cdk_deployer_singleton_factory.ts

```diff
@@ -2,6 +2,7 @@ import { BackendIdentifier } from '@aws-amplify/plugin-types';
 import { CDKDeployer } from './cdk_deployer.js';
 import { CdkErrorMapper } from './cdk_error_mapper.js';
 import { BackendLocator } from '@aws-amplify/platform-core';
+import { PackageManagerControllerFactory } from '@aws-amplify/cli-core';
```

Comment:
In addition.
This will trigger this check
https://github.com/aws-amplify/amplify-backend/blob/40298a493099f8f085d79fdcb705dc24b15a2198/scripts/check_dependencies.ts#L10-L15

Reply:
This would be solved by the discussion here: https://github.com/aws-amplify/amplify-backend/pull/947#discussion_r1465546808